ansible_collections/arista/cvp/molecule/cv_device_v3/test_cv_device_v3.yaml

```diff
@@ -0,0 +1,308 @@
+---
+- name: Test cv_device_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    CVP_DEVICES_MOVE_DEVICE:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
+
+    CVP_DEVICES_PARENT_CONTAINER_NONE:
+      - fqdn: leaf1
+        parentContainerName: None
+
+    CVP_DEVICES_PARENT_CONTAINER_SAME:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
+    CVP_DEVICES_STATE_ABSENT:
+      - fqdn: leaf2
+        parentContainerName: ""
+
+    CVP_DEVICES_FACT0RY_RESET:
+      - fqdn: leaf2
+        parentContainerName: ""
+
+    CVP_DEVICES_PROVISIONING_RESET:
+      - fqdn: leaf3
+        parentContainerName: ""
+
+    CVP_DEVICES_BUNDLE_ATTACHED:
+      - serialNumber: JPE504a004ea054
+        parentContainerName: L2_Leaf
+        configlets:
+          - 'AVD_Ipmi3'
+        imageBundle: leaf_bundle
+
+    CVP_DEVICES_BUNDLE_DETACHED:
+      - serialNumber: JPE504a004ea054
+        parentContainerName: L2_Leaf
+        configlets:
+          - 'AVD_Ipmi3'
+
+    CVP_CONFIGLETS:
+      configlet1: '! This is first configlet'
+      configlet2: '! This is second configlet'
+
+    CVP_DEVICES_APPLY_CONFIGLET:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
+        configlets:
+          - 'configlet1'
+          - 'configlet2'
+
+    CVP_DEVICES_DETACH_CONFIGLET:
+      - fqdn: leaf1
+        parentContainerName: "Leaf"
+
+    CVP_DEVICES_DEPLOY:
+      - fqdn: leaf3    # leaf1 must be in undefined container
+        parentContainerName: "pod2"
```

Comment:
This container doesn't exist on ATD. Please create container and then deploy it there.

Reply:
fixed